x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/nio/SSLDriverTests.java

```diff
@@ -163,21 +165,22 @@ public void testHandshakeFailureBecauseProtocolMismatch() throws Exception {
 
         final String[] serverProtocols;
         final String[] clientProtocols;
-        final String expectedMessage;
+        final Matcher expectedMessageMatcher;
```

Comment:
```suggestion
        private final Matcher expectedMessageMatcher;
```
Can we change it to private? Also, subsequent member variables.

Reply:
Thank you for the feedback, but this is a local variable which doesn't support access modifiers.

Reply:
Ohh my bad 🙈